<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logging</class>
 <widget class="QWidget" name="logging">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="centerWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>382</width>
     <height>282</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/src/wechat2.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="NoBrush">
             <color alpha="128">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="NoBrush">
             <color alpha="128">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="NoBrush">
             <color alpha="128">
              <red>166</red>
              <green>166</green>
              <blue>166</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(166,166,166);</string>
        </property>
        <property name="text">
         <string>myIm</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>账号 :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>密码 :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_accout">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>80</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	border-style: none;
	border-radius: 5px;	
	border-right:1px solid #3498db;
	border-top:1px solid #3498db;
	border-left:1px solid #3498db;
	border-bottom:1px solid #3498db;
	min-height: 20px;
}
QLineEdit:focus {
    border: 2px solid #3498db;
}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>130</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	border-style: none;
	border-radius: 5px;	
	border-right:1px solid #3498db;
	border-top:1px solid #3498db;
	border-left:1px solid #3498db;
	border-bottom:1px solid #3498db;
	min-height: 20px;
}
QLineEdit:focus {
    border: 2px solid #3498db;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_login">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>200</y>
          <width>112</width>
          <height>34</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	
	color: rgb(255, 255, 255);
	border-style: solid;
	border-radius: 5px;	
	background-color: rgb(7,193,96);
}

QPushButton:hover{
	background-color: rgb(56,205,127);
}

QPushButton:pressed {
	background-color: rgb(7,193,96);
}</string>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_regist">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>200</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: rgb(166,166,166);
	border-style: none;
}
QPushButton::hover {
	color: rgb(67,129,202);
}</string>
        </property>
        <property name="text">
         <string>注册账号</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_hide">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>31</width>
          <height>34</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: rgb(166,166,166);
	border-style: none;
}
QPushButton::hover {
	color: rgb(67,129,202);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/src/min.png</normaloff>:/src/min.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_close">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>10</y>
          <width>21</width>
          <height>34</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: rgb(166,166,166);
	border-style: none;
}
QPushButton::hover {
	color: rgb(67,129,202);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/src/close.png</normaloff>:/src/close.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_seePassword">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>140</y>
          <width>20</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/src/eyes_open.png</normaloff>
          <normalon>:/src/eyes_open.png</normalon>:/src/eyes_open.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="6" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="7">
         <widget class="QPushButton" name="pushButton_close2">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(166,166,166);
	border-style: none;
}
QPushButton::hover {
	color: rgb(67,129,202);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/src/close.png</normaloff>:/src/close.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>账号</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>确认密码</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="3">
         <widget class="QLineEdit" name="lineEdit_accout2">
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	border-style: none;
	border-radius: 5px;	
	border-right:1px solid #3498db;
	border-top:1px solid #3498db;
	border-left:1px solid #3498db;
	border-bottom:1px solid #3498db;
	min-height: 20px;
}
QLineEdit:focus {
    border: 2px solid #3498db;
}</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLabel" name="label_10">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="NoBrush">
               <color alpha="128">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="NoBrush">
               <color alpha="128">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="NoBrush">
               <color alpha="128">
                <red>166</red>
                <green>166</green>
                <blue>166</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(166,166,166);</string>
          </property>
          <property name="text">
           <string>myIm</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="pushButton_regist2">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(166,166,166);
	border-style: none;
}
QPushButton::hover {
	color: rgb(67,129,202);
}</string>
          </property>
          <property name="text">
           <string>注册</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>:/src/wechat2.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>昵称</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="3">
         <widget class="QLineEdit" name="lineEdit_passwordConfirm">
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	border-style: none;
	border-radius: 5px;	
	border-right:1px solid #3498db;
	border-top:1px solid #3498db;
	border-left:1px solid #3498db;
	border-bottom:1px solid #3498db;
	min-height: 20px;
}
QLineEdit:focus {
    border: 2px solid #3498db;
}</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QPushButton" name="pushButton_return">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(166,166,166);
	border-style: none;
}
QPushButton::hover {
	color: rgb(67,129,202);
}</string>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="5">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2" colspan="3">
         <widget class="QLineEdit" name="lineEdit_password2">
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	border-style: none;
	border-radius: 5px;	
	border-right:1px solid #3498db;
	border-top:1px solid #3498db;
	border-left:1px solid #3498db;
	border-bottom:1px solid #3498db;
	min-height: 20px;
}
QLineEdit:focus {
    border: 2px solid #3498db;
}</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2" colspan="3">
         <widget class="QLineEdit" name="lineEdit_nickname">
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	border-style: none;
	border-radius: 5px;	
	border-right:1px solid #3498db;
	border-top:1px solid #3498db;
	border-left:1px solid #3498db;
	border-bottom:1px solid #3498db;
	min-height: 20px;
}
QLineEdit:focus {
    border: 2px solid #3498db;
}</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="pushButton_hide2">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(166,166,166);
	border-style: none;
}
QPushButton::hover {
	color: rgb(67,129,202);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/src/min.png</normaloff>:/src/min.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
